docker/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM debian:buster-slim
+ARG SCALAFMT_VERSION
+# TODO: (bsolomon1124) I've tried to use Alpine but gives problems at runtime
```

Comment:
I added a musl build here https://github.com/scalameta/scalafmt/pull/2041
Now you can try an alpine image